pkgs/by-name/li/libansilove/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, gd
+,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libansilove";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "ansilove";
+    repo = "libansilove";
+    rev = version;
```

Use the new `tag` field for tagged releases - `rev` should only be used for commit hashes
```suggestion
    tag = version;
```